spec.json

```diff
@@ -344,6 +346,7 @@
         },
         "largeMotor": {
             "friendlyName": "Large Motor",
+            "systemDeviceNameConvention": "motor{0}",
```

Comment:
@WasabiFan - I will defer to your judgement on this - I agree with pick one and stick with it